### PR TITLE
add announce for week 10

### PR DESCRIPTION
**Pull request by butlerx**

### CODE REVIEW

File: source/_posts/2016-week-10-announce.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+So you want to buy a pi but people keep asking you, 'what can you do with that?' - Well, we got pis and now you need projects.
+Drop by and hear the ever-glowing helpdesk give you project ideas and a basis to get you started.
+
+## RedBrick Secret Santa:
+RedBrick is doing a secret santa (you're anonymously given the name of a person to buy a gift for
+and someone else is made to buy you a gift) the gifts will be exchanged at our Christmas event in
+the last week of the semester - where maybe if you're all extra good, Santa Claus will make an appearence.
+If you want to sign up to the secret santa the link is: events.redbrick.dcu.ie/christmas
+
+Here’s our social media in case you miss us:
```
Comment (VoyTechnology) [inline source/_posts/2016-week-10-announce.md]: Can this be generated?

File: source/_posts/2016-week-10-announce.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+helping alleviate its effects, paired with some tips on how to combat it you'll hopefully leave this talk with an arsenal to stave off burnout
+as you go through life.
+
+## Helpdesk: Raspberry Pi Projects
+
+- When: Wednesday 23rd, 3-4pm
+- Where: LG25
+- More information here: https://www.facebook.com/events/191242068000680/
+
+So you want to buy a pi but people keep asking you, 'what can you do with that?' - Well, we got pis and now you need projects.
```
Comment (VoyTechnology) [inline source/_posts/2016-week-10-announce.md]: Capitalize `Pi`?

File: source/_posts/2016-week-10-announce.md (hunk 1 above)
Comment (butlerx) [inline source/_posts/2016-week-10-announce.md]: it could but a problem for a different day

File: source/_posts/2016-week-10-announce.md (hunk 2 above)
Comment (butlerx) [inline source/_posts/2016-week-10-announce.md]: i just commit what im given

**Pull request merged**